<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>AltVsTime</class>
 <widget class="QWidget" name="AltVsTime" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Altitude vs. Time</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget2" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Sources</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="3" column="0" >
          <widget class="QLabel" name="textLabel4" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Dec:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2" >
          <widget class="KLineEdit" name="nameBox" >
           <property name="toolTip" >
            <string>Name of plotted object</string>
           </property>
           <property name="whatsThis" >
            <string>There are two ways to use this field:  

1. Simply type the name of a known object and press the "Plot" button (or press Enter).  The object's Altitude vs. Time curve will be added to the plot, and its coordinates will be displayed below.

2. Type a name label for a custom object.  You will also need to specify the RA and Dec coordinates of the object, and then press the "Plot" button to add its curve to the plot.</string>
           </property>
           <property name="trapEnterKeyEvent" stdset="0" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1" >
          <widget class="KLineEdit" name="epochName" >
           <property name="minimumSize" >
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>80</width>
             <height>32767</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="textLabel1" >
           <property name="text" >
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item rowspan="2" row="1" column="0" >
          <widget class="QLabel" name="textLabel2" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>RA:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="textLabel3" >
           <property name="text" >
            <string>Epoch:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2" >
          <widget class="dmsBox" name="raBox" />
         </item>
         <item rowspan="2" row="2" column="1" colspan="2" >
          <widget class="dmsBox" name="decBox" />
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>0</number>
         </property>
         <item>
          <widget class="KPushButton" name="browseButton" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>1</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Find Object...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="addButton" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>1</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Plot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="clearFieldsButton" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>1</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Clear Fields</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="clearButton" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>1</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Clear List</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="K3ListBox" name="PlotList" >
         <property name="minimumSize" >
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="vScrollBarMode" >
          <enum>Q3ScrollView::AlwaysOn</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Date &amp;&amp; Location</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="textLabel5" >
           <property name="text" >
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ExtDateEdit" name="dateBox" >
           <property name="minimumSize" >
            <size>
             <width>90</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>257</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="KPushButton" name="cityButton" >
           <property name="text" >
            <string>Choose City...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel7" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Long.:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="dmsBox" name="longBox" />
         </item>
         <item>
          <widget class="QLabel" name="textLabel6" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Lat.:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="dmsBox" name="latBox" />
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="updateButton" >
           <property name="text" >
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>K3ListBox</class>
   <extends></extends>
   <header>k3listbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>ExtDateEdit</class>
   <extends></extends>
   <header>extdatetimeedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>dmsBox</class>
   <extends>KLineEdit</extends>
   <header>widgets/dmsbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends></extends>
   <header>klineedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends></extends>
   <header>kpushbutton.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget2</tabstop>
  <tabstop>nameBox</tabstop>
  <tabstop>epochName</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>clearFieldsButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>PlotList</tabstop>
  <tabstop>cityButton</tabstop>
  <tabstop>updateButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
